src/HttpServer.ts

```diff
@@ -99,6 +110,18 @@ export class HttpServer {
         res.send(status);
     }
 
+    async processRootHandler(method: string, params: any) {
+        if (AVAILABLE_METHODS.includes(method)) {
+            // @ts-ignore
```

Comment:
prefer casting to ignoring rules

Reply:
How about adding the index signature to RelayServer.ts?

`[key: string]: any;`

Reply:
updated